src/components/DashBoard/SideBar/SideBarAdmin.jsx

```diff
@@ -45,15 +47,19 @@ export const SideBarAdmin = ({logout,updateSite}) => {
             <FontAwesomeIcon icon={faSignal} />
             <div className='text'>Estadísticas</div>
         </li>
-        <p className='centerP'>Usuario</p>
+        <p className='centerP text'>Usuario</p>
         <li onClick={() => updateSite("profile")}>
             <FontAwesomeIcon icon={faCircleUser} />
             <div className='text'>Perfil</div>
         </li>
-        <li onClick={logout}>
+        {
+            screen > 768 ? 
+            <li onClick={logout}>
             <FontAwesomeIcon icon={faArrowRightFromBracket}/>
-            <div className='text'>Cerrar Sesión</div>
-        </li>
+            <div>Cerrar Sesión</div>
+            </li> :
+            <></>
+        }
```

Comment:
Here please use classes to display or not that icon, the window.screen doesn't always work

Reply:
Do the same on all Sidebars